File: app/src/main/java/com/base/and/ui/TabDemoFragment.java
package com.base.and.ui;

import androidx.fragment.app.Fragment;

import com.base.and.base.BaseTabFragment;

public class TabDemoFragment extends BaseTabFragment {
    @Override
    protected String getTitle() {
        return "tab页面";
    }

    @Override
    protected String[] getTabNames() {
        return new String[]{"测试1", "测试2", "测试3"};
    }

    @Override
    protected Fragment getTabItem(int position) {
        return Fragment.instantiate(getContext(), MineFragment.class.getName());
    }
}

File: app/src/main/java/com/jeahwan/origin/ui/TabDemoFragment.kt
package com.jeahwan.origin.ui

import androidx.fragment.app.Fragment
import com.jeahwan.origin.base.BaseTabFragment
import com.jeahwan.origin.ui.MineFragment

open class TabDemoFragment : BaseTabFragment() {
    override val title: String
        protected get() = "tab页面"

    override fun getTabNames(): Array<String> {
        return arrayOf("测试1", "测试2", "测试3")
    }

    override fun getTabItem(position: Int): Fragment {
        return instantiate(context!!, MineFragment::class.java.name)
    }
}